ops-bedrock/docker-compose.yml

```diff
@@ -269,3 +269,7 @@ services:
 #      - "9090:8080"
 #    volumes:
 #      - op_log:/op_log:ro
+
+networks:
+  default:
+    name: local-network
```

Comment:
`local-network` probably isn't the greatest of names? Maybe `bedrock-devnet` or something to make it a bit more identifiable in a `docker network ls`?

Reply:
Fixed